lms/djangoapps/verify_student/models.py

```diff
@@ -642,11 +642,14 @@ def upload_photo_id_image(self, img_data):
         Yes, encoding it to base64 adds compute and disk usage without much real
         benefit, but that's what the other end of this API is expecting to get.
         """
-        # Skip this whole thing if we're running acceptance tests or if we're
-        # developing and aren't interested in working on student identity
-        # verification functionality. If you do want to work on it, you have to
-        # explicitly enable these in your private settings.
+        # Skip this whole thing except 'photo_id_key' which is required if
+        # we're running acceptance tests or if we're developing and aren't
+        # interested in working on student identity verification functionality.
+        # If you do want to work on it, you have to explicitly enable these in
+        # your private settings.
         if settings.FEATURES.get('AUTOMATIC_VERIFY_STUDENT_IDENTITY_FOR_TESTING'):
+            self.photo_id_key = 'fake-photo-id-key'
```

Comment:
Please add comment that this fake photo id key will set only for initial verifications